airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -222,6 +220,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     final JobNotifier jobNotifier = new JobNotifier(configs.getWebappUrl(), configRepository);
 
     if (configs.getWorkerEnvironment() == Configs.WorkerEnvironment.KUBERNETES) {
+      KubePortManagerSingleton.warnIfInsufficientPorts(Integer.parseInt(configs.getSubmitterNumThreads()));
```

Comment:
@jrhizor I'm thinking a warn is sufficient. Or do you think we should error out?

Reply:
good idea. let me see how to manage that